content/blog/2024-06-20-why-rse.md

```diff
@@ -0,0 +1,100 @@
++++
+title = "Small steps towards sustainable and open research"
+
+[extra]
+
+author = "Aurélie Vancraeyenest, Samantha Wittke"
```

Comment:
```suggestion
author = "Aurélie Vancraeyenest, Samantha Wittke, Konstantin Stadler"
```
@konstantinstadler please confirm if you want to be listed here as an author :)

Reply:
Thanks for that! Very much like that and fully support the points of the blog post